app/repositories/EncryptedPersonalDetailsValidationRepository.scala

```diff
@@ -134,4 +134,11 @@ class EncryptedPersonalDetailsValidationRepository @Inject()(
         }
       }
   }
+  def clear(nino: String): Future[Boolean] = {
```

Comment:
Is it good to move this method implementation to its trait because we are not encrypting nino and this implementation is the same for both encrypted and non-encrypted repos

Reply:
As above.